sabnzbd/config.py

```diff
@@ -380,7 +380,7 @@ def __init__(self, name, values):
         self.enable = OptionBool(name, 'enable', True, add=False)
         self.optional = OptionBool(name, 'optional', False, add=False)
         self.retention = OptionNumber(name, 'retention', add=False)
-        self.ssl_type = OptionStr(name, 'ssl_type', 't1', add=False)
+        self.ssl_type = OptionStr(name, 'ssl_type', 'v23', add=False)
```

Comment:
I think this is wrong (both original and new code).
It should be empty so that the highest level will automatically be used.
The same goes for the other two places.